recipe/meta.yaml

```diff
@@ -1,46 +1,91 @@
 {% set name = "pyFFTW" %}
 {% set version = "0.13.1" %}
-{% set sha256 = "09155e90a0c6d0c1f2d1f3668180a7de95fb9f83fef5137a112fb05978e87320" %}
+{% set sha256 = "ffc1b3e7f4f74ee38716d668d7606b5fbfded58343655c160c3644fc4acf5d55" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
   sha256: {{ sha256 }}
   patches:
-    - no-system-include.patch
+    - patches/0001-cython3-updates-and-test-build-linker-flags.patch
+    - patches/0002-no-system-include.patch
 
 build:
   number: 0
-  skip: true  # [py<39 and arm64]
+  # fftw not available for s390x
+  # versioneer not updated for py312
+  skip: true  # [py<38 or py>311 or s390x]
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
     - {{ compiler('c') }}
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
-    - cython
+    - cython >=3
     - python
     - pip
-    - numpy
-    - fftw  # [win]
-    - fftw-static  # [not win]
+    - setuptools
```

Comment:
probably not going to make a difference, but whatever it's worth, this is the upstream requirement on `setuptools`
https://github.com/pyFFTW/pyFFTW/blob/965b82f70478e5a2586202243cd645bbb3ba15c9/pyproject.toml#L169

Reply:
I am verifying this again, but I remember that it was not available maybe on py312, and since it was working with the recent version, I've removed the pin completely

Reply:
It does work, my bad. Thanks for bringing this up.